Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM ubuntu:18.04
+
+# Set up the non-root user
+RUN apt-get update \
+    &&  apt-get -y install sudo \
+    && useradd -ms /bin/bash user && echo "user:user" | chpasswd && adduser user sudo
+
+ADD /sudoers.txt /etc/sudoers
```

Comment:
This allows you to use `sudo` without a password while inside the container, which is necessary for building via `sudo python3 ./build/setup.py install` because the `user` user doesn't have access to the `/usr/local` directory (where this installs)

Reply:
Fair enough.